docs/httpproxy.md

```diff
@@ -58,7 +58,7 @@ spec:
 **Line 6-7**: The presence of the `virtualhost` field indicates that this is a root HTTPProxy that is the top level entry point for this domain.
 The `fqdn` field specifies the fully qualified domain name that will be used to match against `Host:` HTTP headers.
 
-**Lines 8-9**: Each HTTPProxy must have one or more `routes`, each of which must have a condition to match against (e.g. `prefix: /`) and then one or more `services` which will handle the HTTP traffic.
+**Lines 8-9**: Each HTTPProxy **must** have one or more routes, each of which **must** have one or more services which will handle the HTTP traffic. In addition, each route **may** have one or more conditions to match against.
```

Comment:
> In addition, each route **may** have one or more conditions to match against.

I think this sentence would be better around line 330.

Reply:
There's a line there already:
> Each Route entry in a HTTPProxy may contain one or more conditions.

I can change the emphasis a bit there if you think that's better.

Reply:
Cool, if that note is mentioned ~330 thats perfect.